packages/legacy/core/App/components/views/CredentialCardLogo.tsx

```diff
@@ -1,36 +1,44 @@
 import { Image, StyleSheet, Text, View } from 'react-native'
 import { BrandingOverlay } from '@hyperledger/aries-oca'
-import { CredentialOverlay } from '@hyperledger/aries-oca/build/legacy'
+import { BrandingOverlayType, CredentialOverlay } from '@hyperledger/aries-oca/build/legacy'
 import { useTheme } from '../../contexts/theme'
 import { toImageSource } from '../../utils/credential'
+import { useEffect, useState } from 'react'
 
 type Props = {
   overlay: CredentialOverlay<BrandingOverlay>
+  brandingOverlayType?: BrandingOverlayType
 }
 
-const logoHeight = 80
-const paddingHorizontal = 24
+const CredentialCardLogo: React.FC<Props> = ({
+  overlay,
+  brandingOverlayType = BrandingOverlayType.Branding10,
+}: Props) => {
+  const { TextTheme, CredentialCardShadowTheme } = useTheme()
+  const [logoText, setLogoText] = useState(overlay.metaOverlay?.name ?? overlay.metaOverlay?.issuer ?? 'C')
+  const logoHeight = brandingOverlayType === BrandingOverlayType.Branding10 ? 80 : 48
+  const paddingHorizontal = 24
 
-const CredentialCardLogo: React.FC<Props> = ({ overlay }: Props) => {
-  const { TextTheme } = useTheme()
+  useEffect(() => {
+    if (brandingOverlayType === BrandingOverlayType.Branding11) {
+      setLogoText(overlay.metaOverlay?.issuer ?? 'I')
+    }
+  }, [brandingOverlayType, overlay])
```

Comment:
I think it might be simpler to just make `logoText` a `useMemo` value. And, to shorten it even further since it only seems to be used in one place, include the `.charAt(0).toUpperCase()` transform in that memo too. What do you think?